src/Service/ImageValidatorNAdder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Image;
+use App\Entity\Article;
+use App\Repository\ImageRepository;
+
+class ImageValidatorNAdder
+{
+    public function addImageIf(string $savedImageId, Article $article, ImageRepository $imageRepository): Article
```

Comment:
You could inject Services in Services using dependency injection in the service constructor

Reply:
Kesako?